CONTRIBUTING_EXTERNAL.md

```diff
@@ -0,0 +1,131 @@
+
+There are a number of ways to contribute to this project.
+
+### Report a Bug
+
+If you think you have found a bug in the code or static site, [search our issues list](https://github.com/HHS/grants-equity/issues) on GitHub for any similar bugs. If you find a similar bug, please update that issue with your details.
+
+If you do not find your bug in our issues list, file a bug report. When reporting the bug, please follow these guidelines:
+
+- **Please use the [Bug Report](https://github.com/HHS/grants-equity/issues/new?assignees=octocat&labels=bug&projects=&template=bug_report.yml&title=%5BBug%5D%3A+) issue template** This is populated with information and questions that will help grants.gov developers resolve the issuethe right information
+- **Use a clear and descriptive issue title** for the issue to identify the problem.
+- **Describe the exact steps to reproduce the problem** in as much detail as possible. For example, start by explaining how you got to the page where you encountered the bug and what you were attempting to do when the bug occurred.
+- **Describe the behavior you observed after following the steps** and point out what exactly is the problem with that behavior.
+- **Explain which behavior you expected to see instead and why.**
+- **Include screenshots and animated GIFs** if possible, which show you following the described steps and clearly demonstrate the problem.
+- **If the problem wasn't triggered by a specific action**, describe what you were doing before the problem happened.
+
+### Suggest an Enhancement
+
+If you don't have specific language or code to submit but would like to suggest a change, request a feature, or have something addressed, you can open an issue in this repository.
+
+Please open an issue of type [Feature request](https://github.com/HHS/grants-equity/issues/new?assignees=octocat&labels=enhancement&projects=&template=feature_request.yml&title=%5BFeature+Request%5D%3A+):
+
+In this issue, please describe the use case for the feature you would like to see -, what you need, why you need it, and how it should work. Team members will respond to the Feature request as soon as possible. Often, Feature request suggestions undergo a collaborative discussion with the community to help refine the need for the feature and how it can be implemented.
+
+### Non-Technical Contributions
+
+#### Documentation
+
+To contribute to documentation you find in this repository, feel free to use the GitHub user interface to submit a pull request for your changes. Find more information about using the GitHub user interface for PRs here.
+
+### Contribute to community discussions
+
+> 🚧 Tools and expanding avenues for community engagement are coming soon.
+
+### Sharing your story 
+
+Sharing how you or your organization have used the Grants Equity project is an important way for us to raise awareness about the project and its impact. Please tell us your story by sending us an email at `grants-equity@hhs.gov`.
```

Comment:
```suggestion
Sharing how you or your organization have used the Grants Equity project is an important way for us to raise awareness about the project and its impact. Please tell us your story by [sending us an email at `grants-equity@hhs.gov`](mailto:grants-equity@hhs.gov).
```

May as well make it clickable

Reply:
Alternatively, you can remove the ticks around grants-equity@hhs.gov and markdown automatically converts it into a mailto link. Using the markdown syntax would allow us to add default subject lines, cc's, etc. But if all we're doing is setting it as a mailto item, that may not be necessary.